src/style.css

```diff
@@ -1 +1,69 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+  min-width: 1024px;
+  max-width: 1200px;
+}
+
+body {
+  background-color: #e5e5e5;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  padding: 0 40px 0 50px;
+  align-items: center;
+  justify-content: space-between;
+  background: #fff;
+}
+
+.logo__img {
+  display: block;
+  width: 40px;
+  height: 40px;
+  position: absolute;
```

Comment:
Remove 'position: absolute;' from the .logo__img class, as it's unnecessary when using flexbox.